stylelint-polaris/tests/index.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-console */
+const fs = require('fs');
+const path = require('path');
+
+const stylelint = require('stylelint');
+
+const fixturePath = path.join(__dirname, 'fixture.css');
+const fileContent = fs.readFileSync(fixturePath, 'utf-8');
+
+(async () => {
+  const {results} = await stylelint.lint({
+    config: {
+      extends: '../index.js',
```

Comment:
```suggestion
      extends: path.join(__dirname, '../index.js'),
```
This was needed for me locally